bears/python/PycodestyleBear.py

```diff
@@ -1,15 +1,16 @@
 import sys
+import re
 
 from coalib.bearlib.abstractions.Linter import linter
 from coalib.settings.Setting import typed_list
+from coalib.results.Result import Result
+from coalib.bearlib.aspects import map_setting_to_aspect
+from coalib.bearlib.aspects.Formatting import LineLength
 
 from dependency_management.requirements.PipRequirement import PipRequirement
 
 
-@linter(executable='pycodestyle',
-        output_format='regex',
-        output_regex=r'(?P<line>\d+) (?P<column>\d+) '
```

Comment:
keep the `output_regex` here as a module level variable.

the diff will then show that the regex hasnt changed.